tests/causal_estimators/test_causalml_estimator.py

```diff
@@ -0,0 +1,216 @@
+import pytest
+import subprocess
+import sys
+
+from dowhy import CausalModel
+from dowhy.datasets import linear_dataset
+
+# To skip the tests if we cannot install causalml
+installed_failed = False

```

Comment:
This is a minor thing, but it's always better to name variables semantically based on their meaning. Here the intended meaning is: "causalml installed or not (because you don't need to install if a user already has it)". 

So a better flag name can be, "is_causalml_installed", or something like that, rather than  conveying whether installation failed or not.